core/src/main/java/io/questdb/griffin/FunctionFactoryDescriptor.java

```diff
@@ -196,6 +196,27 @@ public static int validateSignatureAndGetNameSeparator(String sig) throws SqlExc
         return openBraceIndex;
     }
 
+    public static String createSignatureWithSwappedArgs(String name, String signature) throws SqlException {
```

Comment:
we can perhaps have here a reusable StringBuilder, one for all functions processed by the FunctionFactoryCache